README.md

```diff
@@ -218,6 +218,79 @@ Once you collect all information and create a case study, open a pull request. I
 
 A case study becomes publicly available right after merging and rebuilding the website.
 
+# 🌐 Adding Short URLs to the AsyncAPI Website
+
+The AsyncAPI website supports **short URLs** through Netlify's redirect system using the `public/_redirects` file.  
+This feature enables the creation of memorable, easy-to-share links that redirect to longer URLs — improving user experience and making resources more accessible for the community.
+
+Short URLs are useful for:
+- Community resources (Slack, GitHub)
+- Frequently accessed documentation
+- Marketing campaigns and events
+
+---
+
+## ⚙️ How It Works
+
+All redirects are defined in the `public/_redirects` file. This file follows **Netlify’s redirect syntax** and is automatically processed during deployment.
+
+### Existing Examples
+
+| Short URL | Redirects To | Type |
+|------------|---------------|------|
+| `/slack-invite` | Slack invitation link | Temporary (302) |
+| `/modelina` | `/tools/modelina` | Permanent (301) |
+| `/generator` | `/tools/generator` | Permanent (301) |
+| `/cheatsheet` | GitHub cheatsheet | Permanent (301) |
+
+---
+
+## ✅ When to Create a Short URL
+
+- **Community resources:** Slack, GitHub repositories  
+- **Frequently accessed tools:** AsyncAPI Studio, Generator, Modelina  
+- **Docs shortcuts:** Specification, Getting Started, Tutorials   
+- **Events & campaigns:** Conferences
+
+### Avoid Short URLs For
+
+- Frequently changing internal pages  
+- Temporary content without clear end dates  
+- URLs that conflict with existing site routes  
+- Personal or private organization links  
+- Already short, self-explanatory URLs  
```

Comment:
I would move these sections above instead of lines 226-229, as they repeat the same information